src/serializers/JSONSerializer/index.test.ts

```diff
@@ -14,10 +14,12 @@ describe(`when serializing buffers`, () => {
     })
 })
 
-describe(`when serializing undefined objects`, () => {
-    test('should not attempt to serialize', () => {
-        const got = JSONSerializer(undefined)
-        expect(got).toEqual(undefined)
+describe(`when serializing undefined or null objects`, () => {
+    test.each([
+        [undefined, undefined],
+        [null, 'null'],
```

Comment:
@pwmcintyre - I'm so confused as why this is returning a string... I tried stepping through it but I couldn't figure it out..

Reply:
native stringify
```
$ node
Welcome to Node.js v16.15.0.
Type ".help" for more information.
> JSON.stringify(null)
'null'
```